sklearn/metrics/cluster/unsupervised.py

```diff
@@ -256,3 +257,57 @@ def calinski_harabaz_score(X, labels):
     return (1. if intra_disp == 0. else
             extra_disp * (n_samples - n_labels) /
             (intra_disp * (n_labels - 1.)))
+
+
+def davies_bouldin_index(X, labels):
+    """Computes the Davies-Bouldin Index.
+
+    The index is defined as the ratio of within-cluster distances to
+    between-cluster distances.
+
+    Read more in the :ref:`User Guide <davies-bouldin_index>`.
+
+    Parameters
+    ----------
+    X : array-like, shape (``n_samples``, ``n_features``)
+        List of ``n_features``-dimensional data points. Each row corresponds
+        to a single data point.
+
+    labels : array-like, shape (``n_samples``,)
+        Predicted labels for each sample.
+
+    Returns
+    -------
+    score: float
+        The resulting Davies-Bouldin index.
+
+    References
+    ----------
+    .. [1] `Davies, David L.; Bouldin, Donald W. (1979).
+       "A Cluster Separation Measure". IEEE Transactions on
+       Pattern Analysis and Machine Intelligence. PAMI-1 (2): 224-227`_
```

Comment:
Please add an Examples Sexton

Reply:
I'm sorry, I can see there are sections like this in other parts of the doc, but I don't know how to generate the example contents (?)

Reply:
It should just be a couple of lines showing how you would use this function in a simple case.

Reply:
Isn't the doctest example in lines 1625-1637 doing that?

Reply:
Perhaps. I believe it belongs more here, in the API documentation, than in the narrative user guide